MetasysServicesCom/Interfaces/ILegacyMetasysClient.cs

```diff
@@ -268,6 +276,13 @@ public interface ILegacyMetasysClient
         /// <returns></returns>
         void AddAuditAnnotation(string id, string text);
 
+        /// <summary>
+        /// Add many Annotations given a list of requests containing the Id of the Audits and the text of the Annotations.
+        /// </summary>
+        /// <param name="requests">List of BatchRequestParam to specify the id of the audits and the text of the annotations to add.</param>
+        /// <returns>A list of BatchRequestParam with all the specified attributes.</returns>
+        string[] AddAuditAnnotationMultiple(string[] requests);

```

Comment:
The 'MarshalAs' decorator is needed otherwise will cause a VBA error. Please modify as 
'([In, MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 0)] string[] requests)'